### PR TITLE
Refine the GC frame ref flag commit

### PR DESCRIPTION
**Pull request by wenyongh**

### CODE REVIEW

File: core/iwasm/compilation/aot_emit_control.c [hunk 1]
```diff
@@ -173,20 +173,56 @@ clear_frame_locals(AOTCompFrame *aot_frame)
...
+    /* Reset all the value slots from current frame sp for the else
+       branch since they be the same as starting to translate the
+       if branch */
+    for (p = block->frame_sp_begin; p < p_end; p++) {
+        p->dirty = 0;
+        p->value = NULL;
+        p->type = 0;
+        if (aot_frame->comp_ctx->enable_gc) {
+            p->ref = 0;
+            p->committed_ref = 1;
```
Comment (xujuntwt95329) [inline core/iwasm/compilation/aot_emit_control.c:191]: Why not reset to 0?

**Review by xujuntwt95329 — COMMENTED** [review]
LGTM

File: core/iwasm/compilation/aot_emit_control.c (hunk 1 above)
Comment (wenyongh) [reply core/iwasm/compilation/aot_emit_control.c:191]: `committed_ref == 1` means that the ref flag was already committed and the committed value is `0`, as `p->ref + == p->committed_ref`, no need to commit again next time.

And when translating the else branch, the status of frame refs from current frame_sp is same as translating the if branch. Note that for `if` opcode, AOT code will jump to if branch or else branch, here the `p->committed_ref` should be 1.

Comment (xujuntwt95329) [reply core/iwasm/compilation/aot_emit_control.c:191]: Thanks!

**Pull request merged**